tests/page-objects/forms/generic-form.po.js

```diff
@@ -32,6 +40,20 @@ module.exports = {
     expect(reportInvalidMessage.getText()).toEqual('Has errors');
   },
 
+  invalidateReportNative: async () => {
+    const reviewButton = element.all(by.css('.mm-icon-inverse')).get(0);
```

Comment:
I'm not stocked about this selector '.mm-icon-inverse' . 
All of the icons in the actionbar (left and right!) have this class, we're just lucky this is the 1st one. 

I tried using this instead:
```
const reviewButton = element(by.css('.actions .fa-check')).element(by.xpath('..'));
await helper.waitElementToBeClickable(reviewButton);
```

but then the click ends up on the snackbar for some reason.
I guess we could wait for the snackbar div to disappear (5 seconds).